lona/client/input-events.js

```diff
@@ -106,36 +106,34 @@ Lona.LonaInputEventHandler = function(lona_context, lona_window) {
         var input_delay = event_type[0];
 
         // oninput (input delay)
-        if(node.type == 'text' || node.type == 'textarea') {
```

Comment:
We use ``oninput`` and ``onchange`` to receive user input. ``oninput`` is used for data-binding. With this line in place, data-binding is only  available for inputs with type ``text`` and ``textarea``s. The number input has the type ``number``, so only ``onchange`` and no data-binding would be available

Reply:
Hm...
I see that test you've added fails without this change.
But I don't understand how does example from test_project work.

Reply:
I've figured out why test_project worked.
Data binding happened after `onchange` is fired in browser. It happens after focus is changed, and also it happens if I change number value using up/down keys.

So I totally agree with your changes.

Just in case, you can check out [this branch](https://github.com/maratori/lona/tree/input-number-without-js-change). I've dropped first two commits and added one workaround to check my theory in test.

Reply:
Incidentally i did something very similar with a second input before i realized data binding doesn't work on number inputs :) Thanks for researching!